MCprep_addon/spawner/spawn_util.py

```diff
@@ -41,11 +45,298 @@
 COLL_ICON = 'OUTLINER_COLLECTION' if util.bv30() else 'COLLECTION_NEW'
 
 
+class VariationProp:
```

Comment:
I won't go through this entire class, but don't forget to use `env._` for any user facing strings.